.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish to crates.io
+
+on:
+  push:
+    branches:
+      main
+
+jobs:
+  publish:
+    # need to rename environment name after this is merged
+    environment: Autobump version
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Cardinal-Cryptography/AlephBFT'}}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: force publish if version bumped
+        run: |
+          if [ -n "$(git diff HEAD~ -- Cargo.toml | grep '^+version =')" ]; then
```

Comment:
Again, you can just use the `grep` return code.

Reply:
Will change.